spec/support/db_helpers.rb

```diff
@@ -22,9 +22,7 @@ def connection
     if ENV['BUILDKITE']
       'postgres://buildkite-agent:127.0.0.1:5432/'
     else
-      ENV.fetch('BOXEN_POSTGRESQL_URL') {
-        'postgres://127.0.0.1:5432/'
-      }
+      ENV.fetch('BOXEN_POSTGRESQL_URL') { 'postgres://127.0.0.1:5432/' }
```

Comment:
This can use `ENV.fetch('BOXEN_POSTGRESQL_URL', 'postgres://127.0.0.1:5432/')` if you like.